src/test/crimson/seastore/test_seastore.cc

```diff
@@ -84,6 +95,22 @@ struct seastore_test_t :
 	std::move(t)).get0();
     }
 
+    void set_attr_oi(
+      SeaStore &seastore,
+      bufferlist& val) {
+      CTransaction t;
+      t.setattr(cid, oid, OI_ATTR, val);
+      seastore.do_transaction(
+        coll,
+        std::move(t)).get0();
+    }
+
+    SeaStore::attrs_t get_attr_oi(
```

Comment:
doesn't that make the function unusable in "regular" (not a separate thread) seastar context?